blocks/sticky-header/sticky-header.js

```diff
@@ -1,7 +1,24 @@
+import { createTag } from '../block-helpers.js';
+
 export default function decorate($block) {
   const $header = document.querySelector('header');
+  const $stickyHeader = document.querySelector('.sticky-header.block')
+  const $stickyHeaderContents = document.querySelector('.sticky-header div')
```

Comment:
Always avoid using `document` unless there is a very specific need for it. 

For example, above I call `document` to get the page header, but anything that exists within the block can easily be targeted by replacing `document` with the element:

`const $div = $block.firstElement`